File: src/main/webapp/resources/scripts/domain/ChapterCriteria.js
/* 
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
function ChapterCriteria() {
    this.operand;
    this.chapter;
    this.next;
    
    this.create = function(chapter, operand, next) {
        this.chapter = chapter;
        this.operand = operand;
        this.next = next;
        return this;
    };
    
    this.getOperand = function() {
        this.operand;
    };
    
    this.getChapter = function() {
        this.chapter;
    };
    
    this.getNext = function() {
        this.next;
    };
    
    this.setOperand = function(operand) {
        this.operand = operand;
    };
    
    this.setChapter = function(chapter) {
        this.chapter = chapter;
    };
    
    this.setNext = function(next) {
        this.next = next;
        return this;
    };
};

var criteria = ChapterCriteria.create();

File: src/main/java/com/bhageshri/schoolart/helper/ChapterCriteria.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.bhageshri.schoolart.helper;

import com.bhageshri.schoolart.util.Operand;
import java.io.Serializable;

/**
 *
 * @author mahavir.singh
 */
public class ChapterCriteria implements Serializable {
    private int chapter;
    private Operand operand;
    private ChapterCriteria next;

    public int getChapter() {
        return chapter;
    }

    public void setChapter(int chapter) {
        this.chapter = chapter;
    }

    public Operand getOperand() {
        return operand;
    }

    public void setOperand(Operand operand) {
        this.operand = operand;
    }

    public ChapterCriteria getNext() {
        return next;
    }

    public void setNext(ChapterCriteria next) {
        this.next = next;
    }

    @Override
    public String toString() {
        return "ChapterCriteria{" + "chapter=" + chapter + ", operand=" + operand + ", next=" + next + '}';
    }

    
}
